src/test/ui/issues/issue-16922.nll.stderr

```diff
@@ -12,6 +12,14 @@ LL | fn foo<T: Any>(value: &T) -> Box<Any> {
 LL |     Box::new(value) as Box<Any>
    |     ^^^^^^^^^^^^^^^ lifetime `'static` required
 
-error: aborting due to previous error
+error[E0621]: explicit lifetime required in the type of `value`
+  --> $DIR/issue-16922.rs:14:5
+   |
+LL | fn foo<T: Any>(value: &T) -> Box<Any> {
+   |                       -- help: add explicit lifetime `T` to the type of `value`: `&T T`
```

Comment:
what on earth...?

Reply:
That's super weird. I didn't make any changes to the code when I rebased...